modules/utxo-lib/test/integration_local_rpc/parse.ts

```diff
@@ -81,14 +89,52 @@ function runTestParse(network: Network, txType: FixtureTxType, scriptType: Scrip
       return Buffer.from(getPrevOutput(input).scriptPubKey.hex, 'hex');
     }
 
-    function getPrevOutputs(): [txid: string, index: number, value: number, script: Buffer][] {
-      return parsedTx.ins.map((i) => [getTxidFromHash(i.hash), i.index, getPrevOutputValue(i), getPrevOutputScript(i)]);
+    function getPrevOutputs(): TxbInput[] {
```

Comment:
Maybe rename to `getInputs`?

Reply:
that would suggest `TxInput` which requires some props that we don't have